src/main/java/mmarquee/automation/ElementNotFoundException.java

```diff
@@ -21,4 +21,14 @@
  * Thrown when an element is not found.
  */
 public class ElementNotFoundException extends AutomationException {
+
+    private static final long serialVersionUID = 5575725853478222085L;
+
+    public ElementNotFoundException() {
+        super("Element not fund");
+    }
+
+    public ElementNotFoundException(String name) {
+        super("Element " + name +" not fund");
+    }
```

Comment:
I know the spelling mistake is already there, but can you correct `fund` -> `found`